native/ipc-types.ts

```diff
@@ -22,4 +22,5 @@ export enum RendererMessage {
   GET_VERSION = "GET_VERSION",
   OPEN_PATH = "IPC_OPEN_PATH",
   OPEN_URL = "IPC_OPEN_URL",
+  USERS_GIT_DEFAULT_BRANCH = "USERS_GIT_DEFAULT_BRANCH",
```

Comment:
How about `GET_GIT_GLOBAL_DEFAULT_BRANCH`? It aligns better with `git config --global --get init.defaultBranch` and the fact that all the other variants are verb phrases.

Reply:
Good with me!